velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -448,6 +448,17 @@ VeloxToSubstraitExprConvertor::toSubstraitExpr(
     substraitCastExpr->mutable_input()->MergeFrom(
         toSubstraitExpr(arena, arg, inputType));
   }
+  if (castExpr->nullOnFailure()) {
+    substraitCastExpr->set_failure_behavior(
+        ::substrait::
+            Expression_Cast_FailureBehavior_FAILURE_BEHAVIOR_RETURN_NULL);
+  } else {
+    // It is equivalent to the setting with
+    // Expression_Cast_FailureBehavior_FAILURE_BEHAVIOR_THROW_EXCEPTION.
+    substraitCastExpr->set_failure_behavior(
+        ::substrait::
+            Expression_Cast_FailureBehavior_FAILURE_BEHAVIOR_UNSPECIFIED);
```

Comment:
Why UNSPECIFIED? It might be clearer to use THROW_EXCEPTION.

Reply:
Thanks for your comment! Yes, `THROW_EXCEPTION` is clearer to us. Just corrected.